csm/core/data/models/rgw.py

```diff
@@ -45,6 +45,7 @@ class RgwErrors(Enum):
     EmailExists = "Provided email address exists."
     InvalidCapability = "Attempt to grant invalid admin capability."
     InvalidTenantName = "Invalid tenant name provided."
+    BucketAlreadyExists = "Bucket resource exists."
```

Comment:
This does not look good. Need to discuss. The message should clearly convey that unable to delete IAM user.

Reply:
updated